### DISCUSSION TITLE
pycolmap vs colmap

### DISCUSSION THREAD

**Discussion by shubham-monarch** [colmap/colmap#2254] [General]
I understand that this is a silly question but please bear me. What is the difference between COLMAP and pycolmap? I understand that pycolmap is a python binding for COLMAP? But there must be things which can be done with COLMAP and not pycolmap? And vice-versa?

**Comment by ahojnnes** [comment]
pycolmap binds a subset of the main colmap functionality to be usable directly from python. It allows you to script a custom reconstruction pipeline for research experimentations, etc.

**Reply by shubham-monarch** [reply]
@ahojnnes Thanks for the reply. Is there a reference that contains an exhaustive list of all the colmap functionalities present in pycolmap?

**Reply by shubham-monarch** [reply]
@ahojnnes I think the readme file sums up all the covered functionalities quite nicely. Thanks.
